packages/plugin-bibtex/test/output.json

```diff
@@ -68,7 +68,11 @@
     "label": [[
       {"id": "b", "citation-label": "foo", "type": "book"},
       {"id": "b", "author": [{"given": "foo"}], "type": "book"},
-      {"id": "b", "type": "book"}
+      {"id": "b", "type": "book"},
+      {"id": 123, "type": "book"},
+      {"id": 0, "type": "book"},
+      {"type": "book"},
+      {"id": null, "type": "book"}
```

Comment:
Any idea why the syntax highlighter is representing this as "invalid.illegal"? The tests are passing so it should be valid JSON...

![Screenshot_20211129_202112](https://user-images.githubusercontent.com/14018963/143929384-c158b73a-43e1-4783-b6f1-66aa649ddc55.jpg)

Reply:
I guess that is a bug in the syntax highlighter of github. My editor highlights this as everything else.
According to the [JSON-specification](https://www.json.org/json-en.html), `null` is definitely valid here.

One could argue, that `null` is not valid according to the CSL-Schema, as `id` has to be a `string` or `number` and is required. My idea here was, that it will happen that `null` somehow ends up there, so there should be a test for it.

Reply:
> One could argue, that null is not valid according to the CSL-Schema, as id has to be a string or number and is required. My idea here was, that it will happen that null somehow ends up there, so there should be a test for it.

Yes that's fine. It should never happen when using the `Cite()` constructor because `Cite#format()` [runs the validator before](https://github.com/citation-js/citation-js/blob/efb958674bba4f39abc848053b88e1c7d16ce1ac/packages/core/src/Cite/get.js#L31) but then calling `plugins.output.format()` it does not.